app/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/SubmitReturnsConnector.scala

```diff
@@ -0,0 +1,261 @@
+    val calculatedTaxDue = c.yearToDateLiabilityAnswers.hasEstimatedDetailsWithCalculatedTaxDue.calculatedTaxDue
+
+    DisposalDetails(
+      disposalDate     = c.triageAnswers.disposalDate.value,
+      addressDetails   = addressDetails,
+      assetType        = c.triageAnswers.assetType,
+      acquisitionType  = c.acquisitionDetails.acquisitionMethod,
+      landRegistry     = false,
+      acquisitionPrice = c.acquisitionDetails.acquisitionPrice.inPounds,
+      rebased          = c.acquisitionDetails.rebasedAcquisitionPrice.isDefined,
+      disposalPrice    = c.disposalDetails.disposalPrice.inPounds,
+      improvements     = c.acquisitionDetails.improvementCosts > AmountInPence.zero,
+      percentOwned     = Some(c.disposalDetails.shareOfProperty.percentageValue),
+      acquisitionDate  = Some(c.acquisitionDetails.acquisitionDate.value),
+      rebasedAmount    = c.acquisitionDetails.rebasedAcquisitionPrice.map(_.inPounds),
+      disposalType     = disposalMethod(c),
+      improvementCosts = improvementCosts(c),
+      acquisitionFees  = Some(c.acquisitionDetails.acquisitionFees.inPounds),
+      disposalFees     = Some(c.disposalDetails.disposalFees.inPounds),
+      initialGain      = getInitialGainOrLoss(calculatedTaxDue)._1,
+      initialLoss      = getInitialGainOrLoss(calculatedTaxDue)._2
+    )
+  }
+
+  private def getValueAtTaxBandDetails(calculatedTaxDue: CalculatedTaxDue): Option[List[ValueAtTaxBandDetails]] =
+    calculatedTaxDue match {
+      case _: CalculatedTaxDue.NonGainCalculatedTaxDue => None
+      case g: CalculatedTaxDue.GainCalculatedTaxDue =>
+        Some(
+          List(
+            ValueAtTaxBandDetails(g.taxDueAtLowerRate.taxRate, g.taxDueAtLowerRate.taxableAmount.inPounds()),
+            ValueAtTaxBandDetails(g.taxDueAtHigherRate.taxRate, g.taxDueAtHigherRate.taxableAmount.inPounds())
+          )
+        )
+    }
+
+  private def getTaxableGainOrNetLoss(c: CompleteReturn): (BigDecimal, Option[BigDecimal]) = {
+      val value = c.exemptionsAndLossesDetails.taxableGainOrLoss.getOrElse(
+        c.yearToDateLiabilityAnswers.hasEstimatedDetailsWithCalculatedTaxDue.calculatedTaxDue.taxableGainOrNetLoss
+      )
+
+      if (value < AmountInPence.zero)
+        AmountInPence.zero.inPounds() -> Some(value.inPounds())
+      else value.inPounds()           -> None
+    }
+
+
+  private def getReturnDetails(submitReturnRequest: SubmitReturnRequest): ReturnDetails = {
+    val c                = submitReturnRequest.completeReturn
+    val calculatedTaxDue = c.yearToDateLiabilityAnswers.hasEstimatedDetailsWithCalculatedTaxDue.calculatedTaxDue
+
+    ReturnDetails(
+      customerType          = getCustomerType(submitReturnRequest),
+      completionDate        = c.triageAnswers.completionDate.value,
+      isUKResident          = c.triageAnswers.wasAUKResident,
+      numberDisposals       = c.triageAnswers.numberOfProperties, // make into an Int, sys.error if more than one
```

Comment:
`numberOfDisposals` needs to be an `Int` - since we haven't done multiple disposals yet you could just make it do `sys.error` is the number of properties isn't one